.github/workflows/gradle-publish.yml

```diff
@@ -26,10 +26,24 @@ jobs:
           server-id: github # Value of the distributionManagement/repository/id field of the pom.xml
           settings-path: ${{ github.workspace }} # location for the settings.xml file
 
-      - name: Publish to GitHub Packages
+      - name: Get short commit hash
+        id: commithash
+        run: echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT

```

Comment:
As I see, you don't use `inputs.version`. Either remove it or use it.

Reply:
Actually, I think that I should create a separate task for publishing `usvm-python-runner`. Up until now, the task `gradle-publish` from main and from my branch did different things. In particular, publishing `usvm-python-runner` does not require building of the whole project (even `usvm-python-main`).